.github/workflows/deployment-arm64.yml

```diff
@@ -59,8 +59,8 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v3
         with:
-          java-version: 19
-          distribution: 'liberica'
+          java-version: 20
+          distribution: 'temurin'
```

Comment:
@Siedlerchr please confirm

Reply:
should be fine Using temurin for development as well. I think it was leftover from the last time I experimented with different jdks for signing.